manifest.yml

```diff
@@ -13,3 +13,7 @@ resources:
     path: static/hello-world/build
 app:
   id: ari:cloud:ecosystem::app/6061b1fc-b3fa-4940-a4f9-82ec071d2205
+permissions:
+  scopes:
+    - read:jira-work
```

Comment:
diff apis require different scopes. After calling apis in `src/index.js` file, run `forge lint --fix` to determine missing permissions (if applicable)